src/components/App/App.js

```diff
@@ -1,11 +1,11 @@
 // @flow
-import React, { Component, Fragment } from 'react';
+import React, { PureComponent, Fragment } from 'react';
```

Comment:
The changes in this file is a perf optimization, so that App doesn't re-render unless the selected project is switched